site2/docs/security-athenz.md

```diff
@@ -36,20 +36,13 @@ Note that you can specify any action and resource in step 2 since they are not u
 
 For more specific steps involving the Athenz UI, refer to [Example Service Access Control Setup](https://github.com/AthenZ/athenz/blob/master/docs/example_service_athenz_setup.md#server-provider-domain).
 
-## Enable Athenz authentication on brokers
+## Enable Athenz authentication on brokers/proxies
 
-:::note
-
-When you are using Athenz as an authentication provider, it's highly recommended to use [TLS encryption](security-tls-transport.md) as it can protect role tokens from being intercepted and reused. For more details involving TLS encryption, see [Architecture - Data Model](https://github.com/AthenZ/athenz/blob/master/docs/data_model).
-
-:::
-
-In the `conf/broker.conf` configuration file in your Pulsar installation, you need to provide the class name of the Athenz authentication provider as well as a comma-separated list of provider domain names.
+To configure brokers/proxies to authenticate clients using Authenz, add the following parameters to the `conf/broker.conf` and the `conf/proxy.conf` file. If you use a standalone Pulsar, you need to add these parameters to the `conf/standalone.conf` file, you need to provide the class name of the Athenz authentication provider as well as a comma-separated list of provider domain names.
```

Comment:
Does `provide the class name of the Athenz authentication provider as well as a comma-separated list of provider domain names` also apply to `broker.conf` and `proxy.conf`? The proposed description seems it only applies to `standalone.conf`.

Reply:
This can apply to broker/proxy/standalone.